arangod/RocksDBEngine/RocksDBMethods.h

```diff
@@ -216,6 +224,9 @@ class RocksDBBatchedWithIndexMethods final : public RocksDBMethods {
   rocksdb::Status RollbackToSavePoint() override {
     return rocksdb::Status::OK();
   }
+  rocksdb::Status RollbackToWriteBatchSavePoint() override {
+    return rocksdb::Status::OK();
+  }
   void PopSavePoint() override {}
```

Comment:
Is it a good idea to have these methods just do nothing, rather than throwing a _not implemented_ error?

Reply:
"Not implemented" would actually not be right here in all cases, I guess.
The changes I made try to mimic the behavior of the existing `RollbackToSavePoint()` method, which already had some implementations that simply returned `Status::Ok()`. I will make it clear via code comments that returning `Ok` is not an oversight here.

Reply:
Okay, thanks for the clarification!